ipatests/prci_definitions/nightly_latest_testing.yaml

```diff
@@ -1441,10 +1441,23 @@ jobs:
       args:
         build_url: '{testing-fedora/build_url}'
         update_packages: True
-        test_suite: test_integration/test_ipahealthcheck.py
+        test_suite: test_integration/test_ipahealthcheck.py::TestIpaHealthCheck test_integration/test_ipahealthcheck.py::TestIpaHealthCheckWithoutDNS
         template: *testing-master-latest
         timeout: 4800
```

Comment:
the `topology: *master_1repl` tag is missing, could you add it?

Reply:
@flo-renaud, have corrected the changes.

Reply:
You made an error when fixing the above. `template:` should be `*testing-master-latest` and you need to add `topology: *master_1repl`